### PR TITLE
HTML5 packages upgrade

### PR DESCRIPTION
**Pull request by KDSBrowne**
This PR is a continuation of #6228 with fixes for the reported issues.

* Locales not being displayed when changed
* Tooltip labels not being rendered
* Whiteboard inconsistencies
* Console errors / warnings

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/loading-screen/component.jsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ const LoadingScreen = ({ children }) => (
     <div className={styles.spinner}>
       <div className={styles.bounce1} />
       <div className={styles.bounce2} />
-      <div className={styles.bounce3} />
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/loading-screen/component.jsx]: Why remove that, is for loading has only two bounces?

Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/components/loading-screen/component.jsx]: There was a css error for this style class not existing. Wasn't sure if it was still needed.

Comment (Tainan404) [reply bigbluebutton-html5/imports/ui/components/loading-screen/component.jsx]: it's necessary, stay a strange animation with only two. that class can removed by mistake.

**Comment by MaximKhlobystov** [comment]
@KDSBrowne The version of `postcss-modules-extract-imports` seems to be wrong. That's why Travis CI failed: https://travis-ci.com/bigbluebutton/bigbluebutton/jobs/163421116#L522. I'm getting the same error on my machine when installing packages from scratch.

File: bigbluebutton-html5/package.json [hunk 2]
```diff
@@ -70,19 +71,25 @@
     "yaml": "^1.0.0"
   },
   "devDependencies": {
-    "chai": "~4.2.0",
-    "eslint": "~5.8.0",
+    "axios": "^0.18.0",
```
Comment (antobinary) [inline bigbluebutton-html5/package.json]: Why are `axios` and `dotenv` added?

Comment (antobinary) [reply bigbluebutton-html5/package.json]: also `sha1`, puppeteer, etc - some of these were in a different recent PR.

File: bigbluebutton-html5/package.json [hunk 3]
```diff
@@ -14,7 +14,6 @@
     "precommit": "lint-staged"
```
Comment (MaximKhlobystov) [inline bigbluebutton-html5/package.json]: We need to get rid of this line. It was properly removed here: https://github.com/bigbluebutton/bigbluebutton/pull/6346/commits/44c7e5c10abfcb1f2be897b7ae29be57fbbeeff1#diff-b9183517a2d810804e342afa2421896dL14, but it got back through this merge commit: https://github.com/bigbluebutton/bigbluebutton/pull/6346/commits/462f20d9bbab2626c53bdb7612b78e006f385d10.

**Comment by Tainan404** [comment]
Intl isn't working.
screenshot from 2018-12-07 08-49-15

**Comment by Tainan404** [comment]
I'm unable to join in audio.

**Comment by Tainan404** [comment]
I can't enable close caption.
peek 2018-12-07 08-53

File: bigbluebutton-html5/.meteor/packages [hunk 4]
```diff
@@ -3,24 +3,28 @@
...
+check@1.3.1
+jquery
+
+nathantreid:css-modules@=3.1.4
 rocketchat:streamer
-session
+cfs:power-queue
+cfs:micro-queue
+cfs:reactive-list
+underscore
```
Comment (Tainan404) [inline bigbluebutton-html5/.meteor/packages]: We use Lodash not undescore.
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/package.json#L45

**Pull request merged**